AirCasting/SearchAndFollow/CompleteScreen/Views/CompleteScreenViewModel.swift

```diff
@@ -235,33 +239,33 @@ class CompleteScreenViewModel: ObservableObject {
             }
         }
     }
-
+    
     private func assignValues(with stream: ExternalSessionWithStreamsAndMeasurements.Stream) {
         self.selectedStream = stream.id
         self.selectedStreamUnitSymbol = stream.unitSymbol
     }
-
+    
     private func defineChartRange(with stream: ExternalSessionWithStreamsAndMeasurements.Stream) {
-    guard let separatedSensorName = self.componentsSeparation(name: stream.sensorName) else {
-        Log.error("No sensor name can be extracted from current stream.sensorName")
-        return
-    }
-            (self.chartStartTime, self.chartEndTime) = self.chartViewModel.generateEntries(with: stream.measurements.map({ SearchAndFollowChartViewModel.ChartMeasurement(value: $0.value, time: $0.time) }), thresholds: stream.thresholdsValues, using: ChartMeasurementsFilterDefault(name: separatedSensorName))
+        guard let separatedSensorName = self.componentsSeparation(name: stream.sensorName) else {
+            Log.error("No sensor name can be extracted from current stream.sensorName")
+            return
+        }
```

Comment:
for the future: please don't reformat random code, 99% of this PR is reformatting and is super hard to review. If you feel this needs to be done split the PR into two - one for actual code, one for reformatting only.

Reply:
Sure, thanks for this insight!